@navikt/core/react/src/form-summary/FormSummaryHeading.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { forwardRef } from "react";
+import { Heading, HeadingProps } from "../typography";
+import { OverridableComponent } from "../util/types";
+
+export interface FormSummaryHeadingProps
+  extends React.HTMLAttributes<HTMLHeadingElement> {
+  /**
+   * The heading level.
+   *
+   * @default "3"
+   */
+  level?: HeadingProps["level"];
+  /**
+   * Heading text.
+   */
+  children: React.ReactNode;
+}
+
+export const FormSummaryHeading: OverridableComponent<
+  FormSummaryHeadingProps,
+  HTMLHeadingElement
+> = forwardRef((props, ref) => (
+  <Heading ref={ref} size="medium" level="3" {...props} />
```

Comment:
```suggestion
  <Heading ref={ref} {...props} size="medium" level={props.level} />
```
Bruker "...rest" i de andre vs "...props"

Reply:
Men det er jo ikke noen rest her 😛

Tror ikke `level={props.level}` har noen effekt. Vi tenkte å ha 3 som default level, satte derfor `level="3"`  før `{...props}`. Da vil ev. `level` i props overstyre. Men kanskje det er tydeligere og mindre error-prone å gjøre slik?
```tsx
const FormSummaryHeading = forwardRef(({ level = "3", ...rest }, ref) => (
  <Heading ref={ref} {...rest} size="medium" level={level} />
));
```

Reply:
tenkte det var bare destructure convention > props as is her. endret til å destructure + default assignment, skal funke 🤔